1-org/README.md

```diff
@@ -120,9 +120,9 @@ Run `terraform output cloudbuild_project_id` in the `0-bootstrap` folder to see
    git add .
    git commit -m 'Your message'
    ```
-1. Push your master branch to the new repo.
+1. Push your main branch to the new repo.
    ```
-   git push --set-upstream origin master
+   git push --set-upstream origin main
```

Comment:
@mark1000 lets bundle this change along with the code change after we test it out. I opened https://github.com/terraform-google-modules/terraform-example-foundation/issues/552. Can you revert this bit from the PR?

Reply:
I think I made the correct change. PTAL?